src/main/java/com/backbase/ct/bbfuel/util/ResponseUtils.java

```diff
@@ -0,0 +1,40 @@
+package com.backbase.ct.bbfuel.util;
+
+import static org.apache.http.HttpStatus.SC_BAD_REQUEST;
+
+import com.backbase.buildingblocks.presentation.errors.BadRequestException;
+import io.restassured.response.Response;
+
+public class ResponseUtils {
+
+    private static BadRequestException asBadRequestException(Response response) {
+        return response.then()
+            .extract()
+            .as(BadRequestException.class);
+    }
+
+    private static String getBadRequestMessage(Response response) {
+        return asBadRequestException(response)
+            .getErrors()
+            .get(0)
+            .getMessage();
+    }
+
+    public static boolean isBadRequestExceptionMatching(Response response, String withMessage) {
```

Comment:
I'd suggest to rename `withMessage` to be `messageRegex`

Reply:
fixed